app/src/main/res/menu/bottom_view_menu.xml

```diff
@@ -4,7 +4,7 @@
     <item
         android:id="@+id/fragmentLearn"
         android:enabled="true"
-        android:icon="@drawable/app_ic_rows"
```

Comment:
Please remove `app_ic_row` or use selector for both usages

Reply:
I discussed it with Khalid at that time and we decided to skip the deletion part for now.